aiida/engine/processes/process.py

```diff
@@ -255,6 +299,13 @@ def kill(self, msg=None):
 
     @override
     def out(self, output_port, value=None):
+        """Add output port.
```

Comment:
I would describe this rather as `Register an output`. The port is not the actual thing being added. The port already exists in the process spec and we are merely adding an output whose value is given by `value`, which is first validated against the port